src/Security/OpenIdConfigurationProvider.php

```diff
@@ -56,6 +56,11 @@ class OpenIdConfigurationProvider extends AbstractProvider
      */
     private $responseResourceOwnerId = 'id';
 
+    /**
+     * @var int
+     */
+    private $leeway = 0;
```

Comment:
As far as I can tell you never use this, only the local variable in `validateIdToken` is used?

Reply:
Fixed in 98bf5d0